inc/deprecated/deprecated.php

```diff
@@ -676,11 +676,10 @@ function rocket_insert_critical_css() {
  *
  * @since 2.10
  * @deprecated 2.11
- * @see Rocket_Critical_CSS->insert_load_css()
  * @author Remy Perona
```

Comment:
remove `@author`

Reply:
I don't think we should remove author on deprecated. Should we?